src/unity/schemas/Organization.yml

```diff
@@ -1,23 +1,37 @@
 type: object
 properties:
-  idpeId:
-    type: string
-    description: idpe ID of the org
   id:
-    type: number
-    description: quartz ID of the org
+    type: string
+    description: ID of the org
   name:
     type: string
     description: name of the org
-  region:
+  description:
+    type: string
+    description: the description of the organization
+  creationDate:
+    type: string
+    format: date-time
+    description: the date the organization was created
+  regionCode:
+    type: string
+    description: the code associated with the region
+  regionName:
     type: string
-    description: region of the org
+    description: the name of the region
   provider:
     type: string
     description: provider of the org
-  date:
+  isRegionBeta:
+    type: boolean
+    description: is the region of the organization in beta
+  clusterHost:
     type: string
-    description: date org was created
-  account:
-    $ref: './RelatedAccount.yml'
-required: [account, id, idpeId, region, provider, date]
+    description: the url of the current cluster
+  isActive:
+    type: boolean
+    description: is this the currently active organization in the session?
+  isDefault:
+    type: boolean
+    description: is this the user's default organization in this account?
+required: [id, clusterHost, isRegionBeta, regionCode, regionName, isActive, isDefault]
```

Comment:
Are we going to return `isActive` and `isDefault` from the `GET /orgs/:org_id` endpoint? If not, then should these fields be marked as "not required"?

Reply:
I personally think the `GET /orgs` and `GET /orgs/:idpe_id` should use different schemas, otherwise we can't properly enforce what should and shouldn't be required. Or we end up returning extra information on the list endpoint.